lib/analytics/index.js

```diff
@@ -30,26 +36,17 @@ const analytics = {
         return;
       }
 
-      eventProperties = {
+      const prefix = analytics.getPlatformPrefix();
+
+      if (!prefix) return;
```

Comment:
We should always use [braces around if statements](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#spacing). (We try to follow the Calypso guidelines where appropriate)

Reply:
Thank you! (I was thinking of updating our eslint/prettier rules... They are really not in line with the Calypso guidelines)

Reply:
Sounds good! Although I have gotten used to our prettier format 🤔